internal/app/machined/internal/phase/platform/platform.go

```diff
@@ -56,9 +56,6 @@ func (task *Platform) runtime(r runtime.Runtime) (err error) {
 	}
 
 	if r.Platform().Mode() == runtime.Container {
-		// TODO: add ::1 back once I figure out why bootkube barfs
```

Comment:
We handle injecting this in cluster.go now. Was causing duplicates in machine sans.